CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+This document outlines the contribution workflow, starting from opening an issue, creating a pull request (PR), reviewing, and merging the PR. When working on this project, make sure to follow the [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+Thank you for your contribution.
+
+## New contributor guide 
+
+If you are a new open source contributor, here are some resources you may find useful before providing your first contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/getting-started-with-git/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/using-github/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+**Working on your first pull request?** You can learn how from this free series [How to Contribute to an Open Source Project on GitHub](https://kcd.im/pull-request).
+
+## Create an issue 
+
+If you have any improvement ideas, notice a missing feature or a bug, create a GitHub issue by clicking **Issues -> New issue** in GitHub. Make sure to fill the issue template with the detailed destription of the bug or suggested improvements. Provide proper argumentation and screenshots, if necessary.
+
+If you find any existing issue to work on, you are welcome to open a PR with a fix.
+
+## Open a pull request
+
+If you want to directly contribute to the project, create a pull reguest with the suggested changes. To do so:
+
+1. [Fork the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#fork-an-example-repository).
+
+2. Clone the forked repository to your local machine:
+
+    ```bash
+    git clone git@github.com:{YOUR_GITHUB_USERNAME}/{REPOSITORY_NAME}.git
+    ```
+
+3. Create a new branch:
+
+    ```bash
+    git checkout -b "{BRANCH_NAME}"
+    ```
```

Comment:
I wonder if we need these two points if we provide a link to GitHub docs that explain how to work with forks.

Reply:
No, we can remove it as we have information